src/erlcloud_ec2.erl

```diff
@@ -2936,6 +2946,168 @@ net_if_params(#ec2_net_if{}=X) ->
+        end,
+    P = [
+        {"LogGroupName", LogGroupName},
+        {"ResourceType", RT},
+        {"TrafficType", TT},
+        {"DeliverLogsPermissionArn", DeliverLogsPermissionArn}
+    ] ++ Resources,
+    Params = case ClientToken of
+                none -> P;
+                _ -> P ++ {"ClientToken", ClientToken}
+            end,
+    case ec2_query(Config, "CreateFlowLogs", Params, ?NEW_API_VERSION) of
+        {ok, Doc} ->
+            {ok, [get_text(Item) || Item <- xmerl_xpath:string("/CreateFlowLogsResponse/flowLogIdSet/item", Doc)]};
+        {error, _} = Error ->
+            Error
+    end.
+
+%%
+%% Function for making calls to DeleteFlowLogs action
+%% DescribeFlowLogs Documentation: http://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DeleteFlowLogs.html
+%%
+-spec delete_flow_logs([string()]) -> ok_error(proplist()).
+delete_flow_logs(FlowIDs) ->
+    delete_flow_logs(FlowIDs, default_config()).
+
+-spec delete_flow_logs([string()], aws_config()) -> ok_error(proplist()).
+delete_flow_logs(FlowLogIDs, Config) when is_record(Config, aws_config) ->
+    {Resources, _} = lists:foldl(fun(ID, {Acc, Index}) ->
+                                    I = integer_to_list(Index),
+                                    FlowLogID = "FlowLogId."++I,
+                                    {[{FlowLogID, ID} | Acc], Index+1}
+                            end, {[], 1}, FlowLogIDs),
+
+    case ec2_query(Config, "DeleteFlowLogs", Resources, ?NEW_API_VERSION) of
+        {ok, Doc} ->
+            {ok,[{unsuccessful, get_text("/DeleteFlowLogsResponse/unsuccessful", Doc)}]};
+        {error, _} = Error ->
+            Error
+    end.
+
+%%
+%% Function for making calls to DescribeFlowLogs action
+%% DescribeFlowLogs Documentation: http://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeFlowLogs.html
+%%
+-spec describe_flow_logs() -> ok_error(proplist()).
+describe_flow_logs() ->
+    describe_flow_logs([], [], default_config()).
+
+-spec describe_flow_logs(aws_config()) -> ok_error(proplist());
+                        (ec2_flow_ids()) -> ok_error(proplist()).
+describe_flow_logs(Config)
+    when is_record(Config, aws_config) ->
+    describe_flow_logs([], Config);
+describe_flow_logs(FlowIDs) ->
+    describe_flow_logs(FlowIDs, [], default_config()).
+
+-spec describe_flow_logs(ec2_flow_ids(), filter_list()) -> ok_error(proplist());
+                        (ec2_flow_ids(), aws_config()) -> ok_error(proplist());
+                        (ec2_max_result(), ec2_token()) -> ok_error(proplist(), ec2_token()). 
+describe_flow_logs(FlowIDs, Filter)
+    when is_list(FlowIDs), is_list(Filter) orelse Filter =:= none ->
+    describe_flow_logs(FlowIDs, Filter, default_config());
+describe_flow_logs(FlowIDs, Config)
+    when is_list(FlowIDs), is_record(Config, aws_config) ->
+    describe_flow_logs(FlowIDs, [], Config).
+
+-spec describe_flow_logs(ec2_flow_ids(), filter_list(), aws_config()) -> ok_error(proplist());
```

Comment:
@motobob This follows the same style as describe_instances. Are you proposing completely removing /3 function?